test/unit-tests/vast_test.js

```diff
@@ -443,28 +383,31 @@ describe('ad_manager_vast', function() {
       ads: [vast_ad]
     };
     vastAdManager.initialize(amc);
-    expect(vastAdManager.loadMetadata({"html5_ssl_ad_server":"https://blah",
-      "html5_ad_server": "http://blah"}, {}, content)).to.be(false);
-    initalPlay();
-    expect(vastAdManager.initialPlay()).to.be(true);
+    vastAdManager.loadMetadata({"html5_ssl_ad_server":"https://blah",
+      "html5_ad_server": "http://blah"}, {}, content);
+    expect(amc.timeline.length).to.be(1);
+
+    initialPlay();
+    vastAdManager.initialPlay();
     vastAdManager.onVastResponse(vast_ad,'<VAST></VAST>');
+
     expect(errorType.length > 0).to.be(true);
-    expect(amc.timeline.length).to.be(0);
+    expect(amc.timeline.length).to.be(1);
```

Comment:
Yes. I'd like to revamp these tests but we should do the moving of the AMC into this repo first.  There is too much emulation going on.